doc/src/guides/solving/solve-ode.md

```diff
@@ -0,0 +1,467 @@
+(equation) formatted as, for example, `Eq(y(x), C1*sin(3*x) + C2*cos(3*x))`
+which is equivalent to the mathematical notation $y(x) = C1 \sin(3x) + C2
+\cos(3x)$.
+
+### Extract the Result for One Solution and Function
+
+You can extract the result from an {class}`~.Equality` using the right-hand side
+property {any}`rhs <sympy.core.relational.Relational.rhs>`:
+
+```py
+>>> from sympy import Function, dsolve, Derivative
+>>> from sympy.abc import x
+>>> y = Function('y')
+>>> result = dsolve(Derivative(y(x), x, x) + 9*y(x), y(x))
+>>> result
+Eq(y(x), C1*sin(3*x) + C2*cos(3*x))
+>>> result.rhs
+C1*sin(3*x) + C2*cos(3*x)
+```
+
+### Extract the Result for Multiple Function-Solution Pairs
+
+If you are solving a system of equations with multiple unknown functions,
+{func}`~.dsolve` will return a nested list of equalities, the outer list
+representing each solution and the inner list representing each function. While
+you can extract results by specifying the index of each function ("slicing" each
+solution), we recommend an approach which is robust with respect to function
+ordering. The following converts each solution into a dictionary so you can
+easily extract the result for the desired function. It uses standard Python
+techniques such as a loops or comprehensions, in a nested fashion.
+
+```py
+>>> from sympy import symbols, Eq, Function, dsolve
+>>> y, z = symbols("y z", cls=Function)
+>>> x = symbols("x")
+>>> eqs = [Eq(y(x).diff(x)**2, z(x)**2), Eq(z(x).diff(x), z(x))]
+>>> solutions = dsolve(eqs, [y(x), z(x)])
+>>> solutions
+[[Eq(y(x), C1 - C2*exp(x)), Eq(z(x), C2*exp(x))], [Eq(y(x), C1 + C2*exp(x)), Eq(z(x), C2*exp(x))]]
+>>> solutions_list = [] # nested list approach
+>>> for solution in solutions:
+...     solution_dict = {}
+...     for fn in solution:
+...             solution_dict.update({fn.lhs: fn.rhs})
+...     solutions_list.append(solution_dict)
+>>> solutions_list
+[{y(x): C1 - C2*exp(x), z(x): C2*exp(x)}, {y(x): C1 + C2*exp(x), z(x): C2*exp(x)}]
+>>> solutions_list = [{fn.lhs:fn.rhs for fn in solution} for solution in solutions]
+>>> solutions_list # nested comprehension approach
+[{y(x): C1 - C2*exp(x), z(x): C2*exp(x)}, {y(x): C1 + C2*exp(x), z(x): C2*exp(x)}]
+>>> solutions_list[0][y(x)]
+C1 - C2*exp(x)
+```
+
+### Work With Arbitrary Constants
+
+You can manipulate arbitrary constants such as `C1`, `C2`, and `C3`, which are
+generated automatically by {func}`~.dsolve`, by creating them as symbols. For
+example, if you want to assign values to arbitrary constants, you can create
+them as symbols and then substitute in their values using
+{meth}`~sympy.core.basic.Basic.subs`:
+
+```py
+>>> from sympy import Function, dsolve, Derivative, symbols, pi
+>>> x, C1, C2 = symbols("x, C1, C2")
+>>> y = Function('y')
+>>> result = dsolve(Derivative(y(x), x, x) + 9*y(x), y(x)).rhs
+>>> result
+C1*sin(3*x) + C2*cos(3*x)
+>>> result.subs({C1: 7, C2: pi})
+7*sin(3*x) + pi*cos(3*x)
+```
+
+### Numerically Solve an ODE in SciPy
```

Comment:
I think this should be a level 2 header. It's not related to "use the solution result".